lib/bittrex/order.rb

```diff
@@ -14,9 +14,11 @@ def initialize(attrs = {})
       @total = attrs['Total']
       @fill = attrs['FillType']
       @limit = attrs['Limit']
-      @commission = attrs['Commission']
+      @commission = (attrs['Commission'] || attrs['CommissionPaid']).to_f
```

Comment:
please guard against a nil value here

Reply:
Can you explain this one further?  It is guarding against nil values.  @commission is guaranteed to be set to 0.0 if both attributes are nil.

Reply:
got it. didn't catch the `to_f` on the end